kamon-system-metrics/src/main/scala/kamon/system/sigar/FileSystemMetrics.scala

```diff
@@ -0,0 +1,25 @@
+package kamon.system.sigar
+
+import kamon.metric.GenericEntityRecorder
+import kamon.metric.instrument.{ Memory, InstrumentFactory }
+import org.hyperic.sigar.{ DiskUsage, FileSystem, Sigar }
+
+class FileSystemMetrics(instrumentFactory: InstrumentFactory) extends GenericEntityRecorder(instrumentFactory) with SigarMetric {
+  val reads = DiffRecordingHistogram(histogram("file-system-reads", Memory.Bytes))
+  val writes = DiffRecordingHistogram(histogram("file-system-writes", Memory.Bytes))
+
+  def sumOfAllFileSystems(sigar: Sigar, thunk: DiskUsage ⇒ Long): Long = {
+    val fileSystems = sigar.getFileSystemList.filter(_.getType == FileSystem.TYPE_LOCAL_DISK).map(_.getDevName).toSet
```

Comment:
I dont know if is necessary call always to sigar.getFileSystem List, I mean maybe with the first call is enough, unless we want to update all the time the info of the devices, just in case that appear new one in runtime. what you think?